tools/manage_modules.php

```diff
@@ -49,52 +49,64 @@
     "SELECT Name FROM modules",
     []
 );
-if(isset($flags['remove'])) {
-    foreach($currentModules as $module) {
+if (isset($flags['remove'])) {
+    foreach ($currentModules as $module) {
         try {
             Module::factory($module);
         } catch(\LorisNoSuchModuleException | \LorisModuleMissingException $e) {
             print "Removing $module\n";
-            if($dryrun) {
+            if ($dryrun) {
                 continue;
             }
             $DB->delete("modules", [ 'Name' => $module]);
         }
     }
 }
 
-if(isset($flags['add'])) {
+if (isset($flags['add'])) {
     addDir(__DIR__ . "/../modules");
     addDir(__DIR__ . "/../project/modules");
 }
 
-function addDir(string $moduledir) {
+/**
+ * Adds a new entry to the `modules` table based on a path supplied in the
+ * $moduledir parameter.
```

Comment:
This comment isn't right.

It adds any subdirectories under $moduledir that are valid modules to the modules table.